pkgs/by-name/ze/zed-open-capture/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  libusb1,
+  hidapi,
+  opencv,
+  cmake,
+  pkg-config,
+  fetchFromGitHub,
+  withExamples ? false,
+}:
+
+stdenv.mkDerivation {
+  pname = "zed-open-capture";
+  version = "unstable-2023-24-19";
```

We prefix unstable versions with latest relese before the commit.
```suggestion
  version = "0.5.0-unstable-2023-24-19";
```

---

Thanks and done.